contracts/hardhat.config.ts

```diff
@@ -54,6 +54,7 @@ let config = {
       allowUnlimitedContractSize: Boolean(
         process.env.ALLOW_UNLIMITED_CONTRACT_SIZE,
       ),
+      hardfork: 'merge',
```

Comment:
required due to upgrade to hardhat 2.17 which defaults to shanghai which causes a test to fail